client/src/components/profile/editprofile.js

```diff
@@ -143,22 +218,34 @@ class EditProfilePopup extends Component {
   }
 
   componentDidMount() {
-    const {fullname, email, gender, country} = this.props;
+    const {fullname, email, gender, country, avatar } = this.props;
+    console.log(avatar.top);
     this.setState({
       fullname,
       email,
+      avatar,
+      top: avatar.top,
```

Comment:
Have these states related to avatar completely cohesive into one object called `avatar` or make the state completely isolated.